### ISSUE TITLE
Error when using server ID instead of full name on v1.12

### ISSUE THREAD

**Issue by jasperfirecai2** [42wim/matterbridge#617]
**Describe the bug**
Error on launch when using server ID instead of full name in v1.12

**To Reproduce**

1. Configure discord bot with server as `Server="ID:<id>"`
2. Run executable

**Expected behavior**
The same output that it should give with the full server name

**Screenshots/debug logs**
With ID: https://pastebin.com/dP62R7Rt
With full name: https://pastebin.com/7AtQbMxc

**Environment (please complete the following information):**
 - OS: Windows
 - Matterbridge version: v1.12.0-17-gfc5e3a6-windows-amd64 (development version) (same results in stable v1.12)
 - If self compiled: N/A

**Additional context**
```toml
[discord.tnsfe]
#You can get your token by following the instructions on
#https://github.com/42wim/matterbridge/wiki/Discord-bot-setup
#If you want roles/groups mentions to be shown with names instead of ID,
#you'll need to give your bot the "Manage Roles" permission.
Token="<token>"
Server="ID:<id>"
WebhookURL="https://discordapp.com/api/webhooks/<webhook>"
ShowJoinPart=true
RemoteNickFormat="{NICK}"

[irc.eac]
Server="irc.the-eac.nl:6667"
Nick="MatterBridge"
RemoteNickFormat="[{PROTOCOL}] <{NICK}> "

[api]
[api.matterlink]
    BindAddress="0.0.0.0:4242" # or listen only to localhost: 127.0.0.1:4242
    #OPTIONAL (no authorization if token is empty)
    Token="<token>"
    Buffer=1000
    RemoteNickFormat="{NICK}"
    ShowJoinPart = true

[[gateway]]
name="gateway1"
enable=true

[[gateway.inout]]
account="discord.tnsfe"
channel="ID:351124727416356864"

[[gateway.inout]]
account="irc.eac"
channel="#Minecraft"

[[gateway.inout]]
account="api.matterlink"
```

**Comment by 42wim** [comment]
`Server="ID:<id>"` is not supported at the moment, or did you got this working on an older version of matterbridge and it fails on v1.12 ?

**Comment by 42wim** [comment]
Just released a v1.12.1 which has your issue fixed.
